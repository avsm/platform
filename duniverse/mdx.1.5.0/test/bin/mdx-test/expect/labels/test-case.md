Testing labels:

```sh dir=foo,non-deterministic=bar
$ ls
```

```sh toto,dir=xxx
$ xxx
```
