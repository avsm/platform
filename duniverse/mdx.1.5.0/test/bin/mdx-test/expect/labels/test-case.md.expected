Testing labels:

```sh dir=foo,non-deterministic=bar
>> bar is not a valid value for label "non-deterministic". Valid values are <none>, "command" and "output".
$ ls
```

```sh toto,dir=xxx
>> "toto" is not a valid label or prefix. Valid labels are "dir", "source-tree", "file", "part", "env", "skip", "non-deterministic", "version" and "require-package" and valid prefixes are "set-" and "unset-".
$ xxx
```
